.github/workflows/ci-cd.yml

```diff
@@ -92,8 +92,6 @@ jobs:
     - name: Install core libraries for build
       run: python -Im pip install build
     - name: Build sdists and pure-python wheel
-      env:
-        PIP_CONSTRAINT: requirements/ci.txt
```

Comment:
Not sure if this removal is okay or not. Since it's a pure python build (thus not installing cython), I don't think this has any effect, but I may be mistaken.